### DISCUSSION TITLE
Gists list doesn't sorting from recently update to last

### DISCUSSION THREAD

**Discussion by westor7** [community/community#16251] [Other Feature Feedback, Questions, & Ideas]
Hello,

I edit a public gists page but on API it doesn’t like that, it doesn’t sorting correctly on the top the most recently updated, any idea if this is an API bug ?

Docs: [Gists - GitHub Docs](https://docs.github.com/en/rest/gists/gists#list-public-gists)

Gists link: [westor7’s gists · GitHub](https://gist.github.com/westor7?direction=desc&sort=updated)
API: [https://api.github.com/users/westor7/gists 1](https://api.github.com/users/westor7/gists)

If you compare the first recently updated item you will see that on API is incorrect.

- Thanks!

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
